.github/workflows/ci.yml

```diff
@@ -36,8 +36,48 @@ jobs:
           version: "${{ matrix.NXF_VER }}"
 
       - name: Run pipeline with test data
-        # TODO nf-core: You can customise CI pipeline run tests as required
-        # For example: adding multiple test runs with different parameters
-        # Remember that you can parallelise this by using strategy.matrix
         run: |
           nextflow run ${GITHUB_WORKSPACE} -profile test,docker --outdir ./results
+  test_profiles:
+    name: Run pipeline with test data
+    # Only run on push if this is the nf-core dev branch (merged PRs)
+    if: "${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/bamtofastq') }}"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        config:
+          [
+            "test",
+            "test_cram",
+            "test_chr",
+            "test_no_bai",
+            "test_no_crai",
+            "test_no_stats",
+            "test_no_qc",
+            "test_collate_fast",
+          ]
+        # Nextflow versions
+        include:
+          # Test pipeline minimum Nextflow version
+          - NXF_VER: "21.10.3"
+            NXF_EDGE: ""
+          # Test latest edge release of Nextflow
+          - NXF_VER: ""
+            NXF_EDGE: "1"
```

Comment:
I don't think we do that anymore, please check https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/.github/workflows/ci.yml

Reply:
Hopefully correct now in #53